services/migrations/gitea_uploader.go

```diff
@@ -983,3 +880,25 @@ func (g *GiteaLocalUploader) Finish() error {
 	g.repo.Status = repo_model.RepositoryReady
 	return repo_model.UpdateRepositoryCols(g.repo, "status")
 }
+
+func (g *GiteaLocalUploader) remapExternalUser(source user_model.ExternalUserMigrated, target user_model.ExternalUserRemappable) (err error) {
+	userid, ok := g.userMap[source.GetExternalID()]
+	tp := g.gitServiceType.Name()
+	if !ok && tp != "" {
+		var err error
+		userid, err = user_model.GetUserIDByExternalUserID(tp, fmt.Sprintf("%v", source.GetExternalID()))
```

Comment:
```suggestion
		userid, err = user_model.GetUserIDByExternalUserID(tp,strconv.Itoa(source.GetExternalID()))
```
If `Sprintf` is needed, use `%d`.

Reply:
Changed to %d (needed because it is int64 not int), thanks for the tip.

---

[forgefriends source](https://lab.forgefriends.org/forgefriends/forgefriends/-/merge_requests/39#note_950)